editor/src/components/canvas/canvas-strategies/look-for-applicable-parent-strategy.tsx

```diff
@@ -1,153 +1,88 @@
-import { rectanglesEqual } from '../../../core/shared/math-utils'
-
-export const lookForApplicableParentStrategy: CanvasStrategy = {
-  id: 'LOOK_FOR_APPLICABLE_PARENT_ID',
-  name: (canvasState, interactionSession, strategyState) => {
-    const defaultName = 'Applicable parent'
-    if (interactionSession == null) {
-      return defaultName
-    }
-
-    const result = lookForParentApplicableStrategy(
-      canvasState,
-      interactionSession,
-      strategyState.startingMetadata,
-      strategyState.startingAllElementProps,
-    )
-
-    if (result == null || result.strategies.length === 0) {
-      return defaultName
-    }
-
-    const patchedCanvasState = patchCanvasStateInteractionTargetPath(
-      canvasState,
-      result.effectiveTarget,
-    )
+import { createEmptyStrategyState, InteractionSession } from './interaction-state'
+
+export const lookForApplicableParentStrategy: MetaCanvasStrategy = (
+  canvasSate,
+  interactionSession,
+  metadata,
+  allElementProps,
+) => {
+  if (interactionSession == null) {
+    return []
+  }
 
-    const fittestStrategy = calcFittestStrategy(
-      result.strategies,
-      patchedCanvasState,
-      interactionSession,
-      strategyState,
-    )
+  const result = lookForApplicableParentStrategyInner(
+    canvasSate,
+    interactionSession,
+    metadata,
+    allElementProps,
+  )
 
-    return fittestStrategy.name(canvasState, interactionSession, strategyState) + '*'
-  },
-  controlsToRender: [
-    {
-      control: DragOutlineControl,
-      key: 'ghost-outline-control',
-      show: 'visible-only-while-active',
-    },
-    {
-      control: ParentOutlines,
-      key: 'parent-outlines-control',
-      show: 'visible-only-while-active',
-    },
-    {
-      control: ParentBounds,
-      key: 'parent-bounds-control',
-      show: 'visible-only-while-active',
-    },
-  ],
-  isApplicable: (canvasState, interactionSession, metadata, allElementProps) => {
-    if (interactionSession == null || interactionSession.activeControl.type !== 'BOUNDING_AREA') {
-      return false
-    }
+  if (result == null || result.strategies.length < 1) {
+    return []
+  }
 
-    const strategies = lookForParentApplicableStrategy(
-      canvasState,
-      interactionSession,
-      metadata,
-      allElementProps,
-    )
-    return strategies != null
-  },
+  return [tweakStrategy(result.strategies[0], result.effectiveTarget, result.componentsInSubtree)]
+}
 
-  fitness: (canvasState, interactionSession, strategyState) => {
-    return lookForApplicableParentStrategy.isApplicable(
-      canvasState,
-      interactionSession,
-      strategyState.startingMetadata,
-      strategyState.startingAllElementProps,
-    )
-      ? 1
-      : 0
-  },
+function tweakStrategy(
+  strategy: CanvasStrategy,
+  effectiveTarget: Array<ElementPath>,
+  componentsInSubtree: Array<ElementPath>,
+): CanvasStrategy {
+  const { controlsToRender } = strategy
```

Comment:
as the canvas controls might use `editorState.selectedViews` this might not be correct, depending on the control

Reply:
I made a note for a follow-up fix